pytorch_lightning/trainer/trainer.py

```diff
@@ -605,6 +607,10 @@ def predict(
             return_predictions: Whether to return predictions.
                 ``True`` by default except when an accelerator that spawns processes is used (not supported).
 
+            ckpt_path: Either ``best`` or path to the checkpoint you wish to use to predict.
```

Comment:
what about having True instead of string to reduce eventual confusion between path and type as string